ccd-definition/CaseEventToFields/CareSupervision/manageDocuments.json

```diff
@@ -25,6 +25,19 @@
     "ShowSummaryChangeOption": "Y",
     "FieldShowCondition": "manageDocument.type=\"FURTHER_EVIDENCE_DOCUMENTS\" AND manageDocument.relatedToHearing=\"Yes\""
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageDocuments",
+    "CaseFieldID": "manageDocumentsNoC2sLabel",
```

Comment:
should be fine to keep, existing behaviour can be maintained in service by always settling hasC2s to YES if toggle not switched on